ReactCommon/react/renderer/components/view/YogaLayoutableShadowNode.cpp

```diff
@@ -333,6 +333,27 @@ void YogaLayoutableShadowNode::updateYogaProps() {
   YGStyle result{baseStyle};
 
   // Aliases with precedence
+  if (!props.inset.isUndefined()) {
+    result.position()[YGEdgeAll] = props.inset;
+  }
+  if (!props.insetBlock.isUndefined()) {
+    result.position()[YGEdgeVertical] = props.insetBlock;
+  }
+  if (!props.insetBlockEnd.isUndefined()) {
+    result.position()[YGEdgeBottom] = props.insetBlockEnd;
+  }
```

Comment:
We should give top/bottom precedence over BlockStart/BlockEnd, to match the resolution Yoga will do when given Start/End vs Left/Right. I.e. cardinal direction wins over flow relative direction. The group below this one can be matched for props which lose precedence to ones already set.

I think all the other values should be fine as is. E.g. “insetInlineStart” is the standard conformant name, so it should clobber RN’s “start”. Horizontal/Vertical and All don’t have existing props, so precedence doesn’t matter for them.

Reply:
got it @NickGerleman, I've just pushed a commit fixing this